packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -29,7 +29,7 @@ function hideModal(): void {
 </script>
 
 {#if displayModal}
-  <Modal name="Share your feedback" on:close={hideModal}>
+  <Modal name="Share your feedback" onclose={hideModal}>
```

Comment:
Not sure this is supported, Modal is our custom component.. not sure Svelte do the migration for us here